docs/patterns/existing-eks-observability-accelerators/existing-eks-nginx-observability.md

```diff
@@ -0,0 +1,181 @@
+# Existing EKS Cluster Nginx Observability Accelerator
```

Comment:
This documentation is much needed but check the [Java Mon PR](https://github.com/aws-observability/cdk-aws-observability-accelerator/pull/76) for the approach which we need to revamp.

Reply:
change document based on Java PR